pkg/storage/s3/s3.go

```diff
@@ -773,11 +773,12 @@ func (d *driver) CreateStorage(cr *imageregistryv1.Config) error {
 
 		// at this stage we are not keeping user tags in sync. as per enhancement proposal
 		// we only set user provided tags when we created the bucket.
-		hasAWSStatus := infra.Status.PlatformStatus != nil && infra.Status.PlatformStatus.AWS != nil
-		if hasAWSStatus {
-			klog.Infof("user provided %d tags", len(infra.Status.PlatformStatus.AWS.ResourceTags))
+		tagsInSpec := make(map[string]*struct{})
+		if infra.Status.PlatformStatus.AWS != nil && len(infra.Status.PlatformStatus.AWS.ResourceTags) != 0 {
+			klog.V(5).Infof("infra.Spec has %d user provided tags", len(infra.Status.PlatformStatus.AWS.ResourceTags))
```

Comment:
The code is looking at the `infra.Status` but in this log message (and on line 780) it says `infra.Spec` - should we update the log messages to say `infra.Status` instead?

Reply:
My Bad 
Fixed it!!